src/build.d

```diff
@@ -800,7 +800,7 @@ auto sourceFiles()
             dirEntries(env["C"], "*.d", SpanMode.shallow)
                 .map!(e => e.name)
                 .filter!(e => !e.canFind("dt.d", "obj.d"))
```

Comment:
Why not `buildPath(env["C"], "obj.d")`

Reply:
ok

Reply:
Oh I was referring to the filter line, because then you wouldn't need ty readd it at the end, no?